### DISCUSSION TITLE
Hoe gaan we definities vaststellen

### DISCUSSION THREAD

**Discussion by rubenvdlinde** [OpenCatalogi/.github#88] [Q&A] [answered]
In de praktijk komt een hoop ruis van de vorige componenten catalogus ook door het niet helder hebben van definities. Dat roept de vraag op hoe gaan we definities vaststellen en vastleggen?

Met betrekking tot vastleggen denk ik dat er een  'about' pagina in open catalogi moet zijn (we zouden daarvoor gebruik kunnen maken van read the docs).

Met betrekking tot definities moeten we in ieder geval het volgende afdekken om ruis te voorkomen
- Applicatie 
- Component
- Proces
- Model

Dat kunnen open deuren zijn, maar die moeten we vastleggen. Mooi voorbeeld is component in de huidige catalogus word hiermee de kleinste los te installeren eenheid van een applicatie gebruikt (e.g. iets dat ik op haven kan installeren) in NL Design hebben we het echter over een grafisch component (wat dan weer technisch een package of library is, herbruikbare code binnen een programeertaal of zelfs framework)

**Comment by matthiasoliveiro** [comment, marked as answer]
Rotterdam doet voorzet van de definities en legt die voor bij de validatie groep.
Na feedback validatie groep verfijnen we de definities waar nodig en stellen we de definities vast samen met de ontwerpgroep.

PO stelt een deadline per definitie wanneer deze gereed zou moeten zijn.

**Reply by RonaldvCortenberghe** [reply]
akkoord
